src/views/Editor.vue

```diff
@@ -233,11 +295,15 @@ export default {
 				const isExisting = (currentId && currentId !== 'new')
 				if (isExisting) {
 					payload.id = currentId
-					const response = await apiClient.put("/datasets/" + currentId, payload)
-
+					const _ = await apiClient.put("/datasets/" + currentId, payload)
```

Comment:
No need for const _ =, should be fine with just await promise

Reply:
ok

Reply:
removed